templates/web/pages/product/includes/edit_history.tt.html

```diff
@@ -6,7 +6,7 @@ <h2 id="history">[% lang('history') %]</h2>
 <ul id="history_list">
   [% FOREACH revision IN revisions %]
   <li>
-    [% revision.date %] - [% display_editor_link(revision.userid) %] [% revision.diffs %] [% IF
+    [% revision.date %] - [% display_editor_link(revision.userid) %] (UUID: [% revision.uuid %]) [% revision.diffs %] [% IF
```

Comment:
when I test it, I get an empty UUID:

![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/8158668/704f916e-f833-494c-ae25-6f9947cee4ab)

That's because we need     my $uuid = $change_ref->{app_uuid};    (instead of {uuid}) in Display.pm

But still, some changes won't have a UUID, so in that case we should add a condition, in order not to display (UUID: ) if there is no UUID.

Reply:
updated, thank you!